packages/forms-web-app/__tests__/unit/controllers/lpa-dashboard/confirm-remove-user.test.js

```diff
@@ -56,13 +56,14 @@ describe('controllers/lpa-dashboard/get-confirm-remove-user', () => {
 	});
 
 	describe('postConfirmRemoveUserUser', () => {
-		it.skip('should remove the user', async () => {
+		it('should remove the user', async () => {
 			getLPAUserFromSession.mockReturnValue(mockUser);
 			getUserById.mockResolvedValue(mockRemoveUser);
 
 			await postConfirmRemoveUser(req, res);
 
 			// todo: expect
```

Comment:
can get rid of todo?

Reply:
Will leave as need to still actually remove the user